### PR TITLE
Fix tasks service for new GitHub service interface 

### PR DESCRIPTION
**Pull request by thinkingserious**
As I fixed the tasks service connection, I discovered a timeout issue as well. This PR fixes both of those things.

### CODE REVIEW

File: services/tasks/project/api/tasks.py [hunk 1]
```diff
@@ -127,32 +139,15 @@ def populate_db():
...
+        issues = get_items(repo, 'issues')
+        items = issues + prs
+        response_object[repo] = items
 
     # post payload to /tasks/init
-    for repo in response_object:
+    for repo in all_repos:
         if len(response_object[repo]) != 0:
             for issue in response_object[repo]:
                 if issue != None:
```
Comment (childish-sambino) [inline services/tasks/project/api/tasks.py]: `repo[9:]` below. eep!

File: services/tasks/project/api/tasks.py [hunk 2]
```diff
@@ -127,32 +139,15 @@ def populate_db():
     ]
 
     response_object = dict()
-
-    # make get request to DX_IP
-    # we will have a list of issues for each repo and we will add that to the json response object
-    # which is a dictionary
-    # issues = {status: success, repo1: [list of issues 1], repo2: [list of issues 2], ...}
-    # response_object = {'status' : 'success'}
     client = Client(host="http://{}".format(os.environ.get('DX_IP')))
```
Comment (childish-sambino) [inline services/tasks/project/api/tasks.py]: `client` no longer needed here. Or should be passed to `get_items`. Don't know how heavy it is.

File: services/tasks/project/api/tasks.py [hunk 3]
```diff
@@ -127,32 +139,15 @@ def populate_db():
...
-            response_prs = client.github.prs.get(query_params=query_params)
-            issues = json.loads(response_issues.body)
-            prs = json.loads(response_prs.body)
-            issues_and_prs = issues + prs
-            response_object[repo] = issues_and_prs
-        except Exception as e:
-            print(e)
+        prs = get_items(repo, 'pull_requests')
+        issues = get_items(repo, 'issues')
+        items = issues + prs
```
Comment (childish-sambino) [inline services/tasks/project/api/tasks.py]: Noice!

File: services/tasks/project/api/tasks.py (hunk 1 above)
Comment (thinkingserious) [reply services/tasks/project/api/tasks.py]: Moved this to a config and made a JSON object that we can extend with additional metadata later.

File: services/tasks/project/api/tasks.py (hunk 2 above)
Comment (thinkingserious) [reply services/tasks/project/api/tasks.py]: Great catch, removed!

**Comment by thinkingserious** [comment]
Hello @thinkingserious,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**